core/modules/wbase/Base.cc

```diff
@@ -373,4 +319,4 @@ void StringBuffer2::_setSize(unsigned size) {
     _bufferSize = size;
 }
 
-}}} // namespace lsst::qserv::wbase
+}}} // namespace
```

Comment:
The counterpoint here for me would be that the redundant info in the comments gives context to the closing braces far away from the opening braces.  So I don't have to go diving back to the top of whatever file and pick through the pre-namespace content and try to match up any nested braces in between to know their intent.

That said, I don't really care that much.  But let's at least be consistent -- if we decide to change this we should do it everywhere in the source, and I'd recommend a separate big-bang commit for that?